### DISCUSSION TITLE
安装@formily/vue时报错，版本不兼容

### DISCUSSION THREAD

**Discussion by ChenLeoXX** [alibaba/formily#1736] [Q&A 2.x] [answered]
node版本：v14.16.0
npm版本：7.11.2
vue版本： 3.1.4
@formily/vue版本：^2.0.0-beta.76
报错信息：
![20210707162345](https://user-images.githubusercontent.com/27530887/124726420-501e5c00-df40-11eb-9d2b-c45580a46a99.jpg)

**Comment by ChenLeoXX** [comment, marked as answer]
使用 --force ，--legacy-peer-deps 可以解决报错，但是不清楚是否影响后续使用，能否告知一下

**Reply by liuweiGL** [reply]
这是 npm7 的问题吧，npm7 对 peerDependencies 的处理不一样了，具体的自己查下吧。

**Reply by ChenLeoXX** [reply]
好的 感谢
